### ISSUE TITLE
Appending _FILE to environment variables not working with podman compose

### ISSUE THREAD

**Issue by den5o** [directus/directus#24184]
### Page

https://docs.directus.io/self-hosted/config-options.html#environment-variable-files

### Describe the Inaccuracy

In https://docs.directus.io/self-hosted/config-options.html#environment-variable-files it is stated that appending `_FILE` will work when used with docker secrets with _any_ environment variable.

It appears that this will not work with any variable, at least not with podman-compose. As you can see from the logs below the database user `testuser` is read form the secret, however the database password is not. I tested the database using DBeaver and the user/pass combination is correct.

```
error: password authentication failed for user "testuser
"
    at Parser.parseErrorMessage (/directus/node_modules/.pnpm/pg-protocol@1.7.0/node_modules/pg-protocol/dist/parser.js:283:98)
    at Parser.handlePacket (/directus/node_modules/.pnpm/pg-protocol@1.7.0/node_modules/pg-protocol/dist/parser.js:122:29)
    at Parser.parse (/directus/node_modules/.pnpm/pg-protocol@1.7.0/node_modules/pg-protocol/dist/parser.js:35:38)
    at Socket.<anonymous> (/directus/node_modules/.pnpm/pg-protocol@1.7.0/node_modules/pg-protocol/dist/index.js:11:42)
    at Socket.emit (node:events:517:28)
    at addChunk (node:internal/streams/readable:368:12)
    at readableAddChunk (node:internal/streams/readable:341:9)
    at Readable.push (node:internal/streams/readable:278:10)
    at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
  length: 105,
  severity: 'FATAL',
  code: '28P01',
  detail: undefined,
  hint: undefined,
  position: undefined,
  internalPosition: undefined,
  internalQuery: undefined,
  where: undefined,
  schema: undefined,
  table: undefined,
  column: undefined,
  dataType: undefined,
  constraint: undefined,
  file: 'auth.c',
  line: '321',
  routine: 'auth_failed'
}

Streaming disconnected
```

**Comment by den5o** [comment]
> Related issues:
> 
>     * **[Secrets in Docker Compose aren't working correctly #17056](https://github.com/directus/directus/issues/17056)**

Triple checked all my secrets. No tabs, spaces or newlines are present. I'm also using podman secrets, not secret files.

**Comment by hanneskuettner** [comment]
```
error: password authentication failed for user "testuser
"
```

Looks like you have an extra newline at the on of your username file?

**Comment by den5o** [comment]
> ```
> error: password authentication failed for user "testuser
> "
> ```
> 
> Looks like you have an extra newline at the on of your username file?

As I mentioned in my reply to the bot, I triple checked my secrets. I recreated and tested them twice more for good measure taking extra care. If that is indeed a new line in the username it is either inserted by podman which I find highly unlikely, or the secret is parsed wrong at runtime?

I should mention that there are a lot of new lines in the log that I removed to reduce its length.
